testProjects/project-dependency-configuration/a/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+	id 'java'
+	id 'java-gradle-plugin'
```

Comment:
A lot of testing projects contains the `java-gradle-plugin`, which looks like can be removed.

Reply:
You're right.  I copied the build defs from `junit5-jupiter-starter-gradle` but they're not required.  I've removed them and pushed the changes